examples/research_projects/rag/distributed_ray_retriever.py

```diff
@@ -134,8 +133,6 @@ def get_tokenizers(cls, retriever_name_or_path, indexed_dataset=None, **kwargs):
 
     @classmethod
     def from_pretrained(cls, retriever_name_or_path, actor_handles, indexed_dataset=None, **kwargs):
-        requires_datasets(cls)
```

Comment:
Why not leave those?

Reply:
Actually, these two functions are outdated and not there anymore.

Instead, now we have, is_faiss_available and is_datasets_available.